posts/2021-03-13-microprofile-config-2.0.adoc

```diff
@@ -31,7 +31,7 @@ Then, it's just 2 steps to launch:
 
 1. You’ll need to tell your build where all the awesome new features are defined. 
 +
-The easiest way to do this it to point to the link:https://search.maven.org/artifact/org.eclipse.microprofile/microprofile/4.0.1/pom[MicroProfile 4.0 Repository] which includes the link:https://search.maven.org/artifact/org.eclipse.microprofile.config/microprofile-config-api/2.0/jar[MicroProfile Config 2.0 API].
+The easiest way to do this it to point to the link:https://search.maven.org/artifact/org.eclipse.microprofile/microprofile/4.0.1/pom[MicroProfile 4.0 API] artifact which includes the link:https://search.maven.org/artifact/org.eclipse.microprofile.config/microprofile-config-api/2.0/jar[MicroProfile Config 2.0 API] artifact.
 +
 - If you’re using link:https://maven.apache.org/[Maven] add the following dependency to your pom.xml:
```

Comment:
Change,
"If you’re using link:https://maven.apache.org/[Maven] add the following dependency to your pom.xml:"
to,
"If you’re using link:https://maven.apache.org/[Maven], add the following dependency to your pom.xml:"